### PR TITLE
vmlatency: Add diagram to readme

### PR DESCRIPTION
**Pull request by ormergi**
Add updated high-level-design diagram to vmlatency checkup readme

https://github.com/ormergi/kiagnose/blob/51eff311b97537fe69ed197f905be825510652ce/docs/images/kubevirt-vm-latency-diagram.svg

Signed-off-by: Or Mergi <ormergi@redhat.com>

### CODE REVIEW

File: checkups/kubevirt-vm-latency/README.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ Currently, the measurement is done using standard [ping](https://en.wikipedia.or
 
 It can ease the maintainability effort required from a cluster administrator by removing the burden of manually testing network connectivity and performance, reduce mistakes and generally save time.
 
+![](../../docs/images/kubevirt-vm-latency-diagram.svg)
```
Comment (orelmisan) [inline checkups/kubevirt-vm-latency/README.md:11]: nit: Please consider adding and alt text and title to the image.
```suggestion
![Block diagram](../../docs/images/kubevirt-vm-latency-diagram.svg "Block diagram")
```

**Review by orelmisan — APPROVED** [review]
Thank you for the PR @ormergi 
The diagram looks great!

File: checkups/kubevirt-vm-latency/README.md (hunk 1 above)
Comment (ormergi) [reply checkups/kubevirt-vm-latency/README.md]: Done

Comment (orelmisan) [reply checkups/kubevirt-vm-latency/README.md]: Thanks

**Review by orelmisan — APPROVED** [review]

**Review by EdDev — APPROVED** [review]
Very nice, thanks!

**Review by RamLavi — APPROVED** [review]
LIKE
/lgtm

**Pull request merged**